signingscript/docker.d/passwords.yml

```diff
@@ -200,6 +200,11 @@ in:
              {"$eval": "AUTOGRAPH_GPG_PASSWORD"},
              ["autograph_gpg"]
             ]
+          - ["https://autograph-external.prod.autograph.services.mozaws.net",
+             {"$eval": "AUTOGRAPH_XPI_USERNAME"},
+             {"$eval": "AUTOGRAPH_XPI_PASSWORD"},
+             ["autograph_xpi", "autograph_xpi_sha1_es256_es384", "autograph_xpi_sha1_es256_ps256", "autograph_xpi_sha1_es256", "autograph_xpi_sha1_ps256"],
```

Comment:
If we use the same formats here as in the stage case below, the latter will not be reachable (see `signingscript.sign.get_autograph_config`).  To have autograph prod and stage xpi signing in adhoc-t, they should get distinct sets of formats.

Reply:
Good catch!

It seems like this function should probably factor in the `keyid` right? Otherwise any formats that the `prod` config has will be unreachable in the `stage` config. If the formats are different, then at least there's *a* format we can test on stage, but we might want to test all of them.

Reply:
I'm not sure.  One thing we could do is have `autograph_stage_xpi*` for stage autograph, and `autograph_xpi*` format using prod.  That'd mirror the existing `autograph_stage_mar384` vs `autograph_mar384` formats.

Reply:
Yeah, I like that. Nice and simple